### ISSUE TITLE
No distribution available.

### ISSUE THREAD

**Issue by MaxTakahashi** [usharesoft/hammr#30]
Hi,

Hammr returns "No distribution available" message, although I can find distributions with UForge Portal UI.
```
[root@localhost ~]# hammr os list --url https://factory.usharesoft.com:443/api -u user -p xxx
Getting distributions for [user] ...
No distributions available
```

Thanks.

**Comment by obourdon** [comment]
I have fixed this in the 3.5.1.3 UForge Python SDK available at
http://www.olivierbourdon.com/USHARESOFT/HAMMR/uforge_python_sdk-3.5.1.3.tar.gz

**Comment by jbremond** [comment]
A UForge Python SDK has been release officialy on Pypi.

I can closed the bug.

Thanks.
Regards,
Joris
